pkg/netstat/format.go

```diff
@@ -166,16 +166,26 @@ func convertUID(uid uint32) (string, error) {
 }
 
 func (o *Output) getNameFromInode(inode uint64) (string, error) {
-	var s strings.Builder
+	var (
+		s   strings.Builder
+		err error
+	)
+
 	pnote := o.ProgCache[int(inode)]
+
 	if pnote.PID == 0 {
-		s.WriteString("-")
+		if _, err = s.WriteString("-"); err != nil {
```

Comment:
It's not clear to me how `WriteString` can fail here?
```go
// WriteString appends the contents of s to b's buffer.
// It returns the length of s and a nil error.
func (b *Builder) WriteString(s string) (int, error) {
	b.copyCheck()
	b.buf = append(b.buf, s...)
	return len(s), nil
}
```

I would rather have something like `_, _ = s.WriteString("-")`

Reply:
This sounds reasonable. This could also be a good solution for the flag parsing issues @rminnich stated.